mojito-functions/reports.R

```diff
@@ -8,9 +8,11 @@ library("reshape")
 library("ztable")
 library("dplyr")
 library("stats")
-
+library("knitr")
+library("jsonlite")
 
 # Report options
+knit_hooks$set(pngquant = hook_pngquant)
```

Comment:
May need to make a comment about the dependency on pngquant CLI  being installed. https://pngquant.org/

Reply:
Yep - good call. Will need to reflect that in the docs too: https://mojito.mx/docs/r-analytics-intro